src/http.ts

```diff
@@ -128,9 +126,6 @@ export async function getJSON<T>(
   }
 
   if (fetchError) {
-    // unfetch uses XMLHttpRequest under the hood which throws
-    // ProgressEvents on error, which don't have message properties
-    fetchError.message = fetchError.message || 'Failed to fetch';
```

Comment:
As we do not use unfetch anymore, this part is dropped and the test updated accordingly